ci.sh

```diff
@@ -174,26 +174,32 @@ if [ "$PROVIDER_NAME" = "trusted-service" ]; then
 fi
 
 if [ "$PROVIDER_NAME" = "coverage" ]; then
-    PROVIDERS="mbed-crypto tpm pkcs11"
-    # Install tarpaulin and start Parsec with it
+    PROVIDERS="mbed-crypto tpm" # pkcs11 not supported because of a segfault when the service stops
```

Comment:
Maybe it would be good to open an issue to reference that and link it here!

Reply:
Yeah, I'll open an issue!